src/app/components/lineage-graph/lineage-graph.component.ts

```diff
@@ -38,12 +40,14 @@ export class LineageGraphComponent implements OnChanges, OnInit {
     public showMiniMap = true;
     public miniMapPosition: MiniMapPosition;
     public graphNodes: Node[];
+    public datasetKind: string;
 
     public readonly DEFAULT_AVATAR_URL = AppValues.DEFAULT_AVATAR_URL;
     public readonly LineageGraphNodeKind: typeof LineageGraphNodeKind = LineageGraphNodeKind;
     public readonly DatasetKind: typeof DatasetKind = DatasetKind;
 
     public ngOnInit(): void {
+        console.log("===>", this.currentDataset);
```

Comment:
I assume this is a leftover? :)
Please remove

Reply:
Removed